backend/UserGroups.sql

```diff
@@ -0,0 +1,41 @@
+-- phpMyAdmin SQL Dump
```

Comment:
I would recommend we create only 1 sql file for the whole backend project.

Reply:
Fixed